### PR TITLE
feat: cosmos defi UI part 2: update translations

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

This updates the translations and removes the now unused old staking ones.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

partially tackles https://github.com/shapeshift/web/issues/2219

## Risk

- We will need to make sure translations are provided for all languages.

## Testing

- None in this PR, compare with part1 to make sure translations are referenced properly.

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2293** <a href="https://app.graphite.dev/github/pr/shapeshift/web/2293" target="_blank">Graphite</a> 
    * **PR #2294** <a href="https://app.graphite.dev/github/pr/shapeshift/web/2294" target="_blank">Graphite</a>  👈
      * **PR #2295** <a href="https://app.graphite.dev/github/pr/shapeshift/web/2295" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/2294?utm_source=stack-comment).

File: src/assets/translations/en/main.json [hunk 1]
```diff
@@ -156,26 +156,21 @@
...
+      "cosmosOverview": {
+        "rewards": "The rewards you earn will accrue automatically.",
+        "cta": "Start Staking",
+        "learnMore": "Learn More",
+        "withdrawals": "Unstaking",
+        "emptyWithdraws": "Pending unstakings will show here.",
+        "availableDate": "available %{date}"
       },
       "foxyOverview": {
         "header": "Deposit FOX to get FOXy to earn",
```
Comment (decentralized-j) [inline src/assets/translations/en/main.json:176]: Request change to 'Deposit FOX to receive and earn FOXy'

Comment (gomesalexandre) [reply src/assets/translations/en/main.json:176]: That's not within the diff of this PR, but that's already how the final string looks like ser ✅

Comment (gomesalexandre) [reply src/assets/translations/en/main.json:176]: That's outside of the scope of this PR, but I'll get a separate one spinned up for ya!

File: src/assets/translations/en/main.json [hunk 2]
```diff
@@ -604,9 +600,23 @@
...
+      "amountToStake": "Amount To Stake",
+      "stakeYour": "Stake your %{opportunity}",
+      "estimatedReturns": "Estimated Yearly Returns*",
+      "estimateDisclaimer": "*Estimates fluctuate with market conditions.",
+      "footerDisclaimer": "By continuing you agree to our ",
+      "footerDisclaimerLink": "terms of service.",
+      "status": {
+        "pending": "Your transaction has been broadcast and is pending confirmation.",
+        "success": "You have successfully staked your %{opportunity}.",
+        "failed": "There was an issue with your stake."
```
Comment (decentralized-j) [inline src/assets/translations/en/main.json:613]: Request change to 'There was an issue with your staking transaction.'

File: src/assets/translations/en/main.json [hunk 3]
```diff
@@ -156,26 +156,21 @@
...
-        "tooltip": {
-          "validator": "The Validator is responsible for processing transactions, storing data and adding blocks to the blockchain.",
-          "gasFees": "Gas fees on %{networkName} are much lower than those on Ethereum Mainnet. This fee is paid directly to the validators that secure and operate the network and is not paid to ShapeShift."
-        }
+      "cosmosOverview": {
+        "rewards": "The rewards you earn will accrue automatically.",
+        "cta": "Start Staking",
+        "learnMore": "Learn More",
+        "withdrawals": "Unstaking",
+        "emptyWithdraws": "Pending unstakings will show here.",
```
[the remark this answers is not in the document]
Comment (gomesalexandre) [reply src/assets/translations/en/main.json]: Done in 9e0db2906bec657c81a74eb3ff5013005d21292f

File: src/assets/translations/en/main.json (hunk 2 above)
Comment (gomesalexandre) [reply src/assets/translations/en/main.json]: Done in b6502b6c3a5ebb85299da6fcab2ca3491697b7ea

**Review by 0xdef1cafe — APPROVED** [review]

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**